validateSql.hs

```diff
@@ -47,12 +50,13 @@ processParams (fname, connstr) = do
 
 checkAllQueries :: Connection -> [ByteString] -> IO ()
 checkAllQueries conn queries = do
-  results <- zipWithM fn queries [B.pack $ "stmt" ++ show x | x <- [1..]]
-  forM_ (zip queries results) $ \case
+  results <- zipWithM fn queries [B.pack $ "stmt" ++ show x | x <- [(1::Int)..]]
+  let cleanResults = filter ((/=) (Left "ignore")) results
```

Comment:
NOTE - this isn't really required in this initial change. It's my personal change to "ignore" quasi quoters with multiple statements.

Initially, I could remove this. In a subsequent PR, I can make it a proper flag.